examples/tutorial/jupyter/README.md

```diff
@@ -15,8 +15,10 @@ please install every package listed in `requirements.txt` file individually with
 
 ## Run Jupyter Notebooks
 
-A Jupyter Notebook example can be run from the current directory as follows:
+A Jupyter Notebook server example can be run from the current directory as follows:
 
 ```bash
-jupyter notebook local/exercise_1.ipynb
+jupyter notebook
 ```
+
+And then navigate to the needed notebook (for example to the `execution/pandas_on_ray/local/exercise_1.ipynb`).
```

Comment:
Jupyter server should be started in the directory containing `img` directory, otherwise notebooks will not capable to access images.

Reply:
Can we use absolute paths, but not relative?

Reply:
Fixed.

Reply:
I see the relative paths. Can we use absolute paths instead?

Reply:
Absolute patrh `examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_1.ipynb` is used now.